Sources/XcbeautifyLib/CaptureGroups.swift

```diff
@@ -1866,3 +1866,368 @@ struct SwiftDriverJobDiscoveryCompilingCaptureGroup: CaptureGroup {
         self.project = project
     }
 }
+
+struct SwiftTestingRunStartedCaptureGroup: CaptureGroup {
+    static let outputType: OutputType = .result
+
+    /// Regular expression to capture the start of a test run.
+    /// $1 = symbol
+    /// $2 = message
+    static let regex = Regex(pattern: #"^(􀟈)\s*(Test run started\.)$"#)
+
+    let symbol: String
```

Comment:
Any reason to capture the symbol here and in the other structs? It doesn't look like you use them to format output. 

Not an issue if we keep them anyway... just curious if you meant to.